pubspec.yaml

```diff
@@ -11,7 +11,10 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_compass: ^0.6.1
-  flutter_map: ^0.13.1
+  flutter_map: 
+    git: 
+      url: "https://github.com/sandeshgyw/flutter_map.git"
+      ref: "13db571"
```

Comment:
Please don't add unpublished Dart packages here. Let's use the official flutter_map.

You can't publish a Dart packages build on top of unpublished repos/packages.